From 8b5864f3efe4e0848ebc83f6abd9a86accb5bd8b Mon Sep 17 00:00:00 2001
From: Hongchao Deng <hdeng@cloudera.com>
Date: Tue, 9 Sep 2014 13:20:05 -0700
Subject: [PATCH 31/94] Revert "ZOOKEEPER-1811. The ZooKeeperSaslClient
 service name principal is hardcoded to "zookeeper"
 (Harsh J via phunt)"

This reverts commit fc459d269159b4a45edbfcad3acf4d5e5253439f.
---
 src/java/main/org/apache/zookeeper/ClientCnxn.java |    9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/src/java/main/org/apache/zookeeper/ClientCnxn.java b/src/java/main/org/apache/zookeeper/ClientCnxn.java
index 059c372..27547b4 100644
--- a/src/java/main/org/apache/zookeeper/ClientCnxn.java
+++ b/src/java/main/org/apache/zookeeper/ClientCnxn.java
@@ -85,9 +85,6 @@ import org.slf4j.LoggerFactory;
 public class ClientCnxn {
     private static final Logger LOG = LoggerFactory.getLogger(ClientCnxn.class);
 
-    private static final String ZK_SASL_CLIENT_USERNAME =
-        "zookeeper.sasl.client.username";
-
     /** This controls whether automatic watch resetting is enabled.
      * Clients automatically reset watches during session reconnect, this
      * option allows the client to turn off this behavior by setting
@@ -943,11 +940,7 @@ public class ClientCnxn {
             setName(getName().replaceAll("\\(.*\\)",
                     "(" + addr.getHostName() + ":" + addr.getPort() + ")"));
             try {
-                String principalUserName = System.getProperty(
-                        ZK_SASL_CLIENT_USERNAME, "zookeeper");
-                zooKeeperSaslClient =
-                    new ZooKeeperSaslClient(
-                            principalUserName+"/"+addr.getHostName());
+                zooKeeperSaslClient = new ZooKeeperSaslClient("zookeeper/"+addr.getHostName());
             } catch (LoginException e) {
                 // An authentication error occurred when the SASL client tried to initialize:
                 // for Kerberos this means that the client failed to authenticate with the KDC.
-- 
1.7.9.5

